refactor_layout.py

```diff
@@ -0,0 +1,1633 @@
+                ("frc/Threads.h", "wpi/system/Threads" + NEW_CC_FILE_SUFFIX),
+                ("frc/Timer.h", "wpi/system/Timer" + NEW_CC_FILE_SUFFIX),
+                ("frc/Tracer.h", "wpi/system/Tracer" + NEW_CC_FILE_SUFFIX),
+                ("frc/WPILibVersion.h", "wpi/system/WPILibVersion" + NEW_CC_FILE_SUFFIX),
+                ("frc/Watchdog.h", "wpi/system/Watchdog" + NEW_CC_FILE_SUFFIX),
+            ],
+        ),
+        (
+            pathlib.Path("wpilibc/src/generated/main/native/include"),
+            [
+                ("frc/StadiaController.h", "wpi/driverstation/StadiaController" + NEW_CC_FILE_SUFFIX),
+                ("frc/PS4Controller.h", "wpi/driverstation/PS4Controller" + NEW_CC_FILE_SUFFIX),
+                ("frc/PS5Controller.h", "wpi/driverstation/PS5Controller" + NEW_CC_FILE_SUFFIX),
+                ("frc/XboxController.h", "wpi/driverstation/XboxController" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMSparkFlex.h", "wpi/hardware/motor/PWMSparkFlex" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/SparkMini.h", "wpi/hardware/motor/SparkMini" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMVictorSPX.h", "wpi/hardware/motor/PWMVictorSPX" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Jaguar.h", "wpi/hardware/motor/Jaguar" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Victor.h", "wpi/hardware/motor/Victor" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/DMC60.h", "wpi/hardware/motor/DMC60" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/VictorSP.h", "wpi/hardware/motor/VictorSP" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMVenom.h", "wpi/hardware/motor/PWMVenom" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Koors40.h", "wpi/hardware/motor/Koors40" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/SD540.h", "wpi/hardware/motor/SD540" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Talon.h", "wpi/hardware/motor/Talon" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Spark.h", "wpi/hardware/motor/Spark" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMTalonSRX.h", "wpi/hardware/motor/PWMTalonSRX" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMTalonFX.h", "wpi/hardware/motor/PWMTalonFX" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMSparkMax.h", "wpi/hardware/motor/PWMSparkMax" + NEW_CC_FILE_SUFFIX),
+            ],
+        ),
+    ]
+
+    # Package changes (and therefor folder moves) for java projects.
+    # Format: (project, original package, new package)
+    JAVA_PROJECT_REPLACMENTS = [
+        ("wpiutil", "edu.wpi.first.util", "org.wpilib.util"),
+        ("wpimath", "edu.wpi.first.math", "org.wpilib.math"),
+        ("wpinet", "edu.wpi.first.net", "org.wpilib.net"),
+        ("datalog", "edu.wpi.first.datalog", "org.wpilib.datalog"),
+        ("command", "edu.wpi.first.wpilibj2.command", "org.wpilib.command"),
```

Comment:
This should use the `org.wpilib.commands2` package name for differentiation with the v3 API (which is already set up to use `org.wpilib.commands3`)

Reply:
I was thinking that should be the case but followed the spreadsheet exactly for the first cut

Reply:
Is the intent for them to be in the same subproject, or will there be a `<root>/command2, <root>/command3`

Reply:
They'll be in different subprojects

Reply:
Note this also needs to be fixed in the build system patch